alexxsanya/ireporter-endpoints
# [#163389277] Creating So More APIs and Remove a build Error

**What does this PR do?**

- Add some more new APIs
- Remove the error of duplication in the requirement file
- Update SQL query [making sttatus NON UNIQUE]

**Description of Task to be completed?**
- Modified the create user API to created admin users
- Added an API to create an intervention incident
- Created the delete user api
- Removed the duplications of configparser from the requirement.txt

**How should this be manually tested?**
- On this Pull request navigate to the files changed tab to see the various files changed

@matthewacha  @alexxsanya @jennizalwango @wabuluka @noahkalyesubula @lizz24 @kalsmic @k7ko @bekeplar @e-ian @masete @Manorlds-Eaglespark @nanfuka @HarrietAkot @PatrickMugayaJoel @kyakusahmed @Anguandia @CandySusan @kabaliisa @bisonlou @HabibSentongo @v1b3m @kdenno @davicken @dtnnml @Araali1 @natalie-abbie @ernesthenry @jamesmudidi

Thanks to @jamesmudidi for the guiding me on how the PR works
